src/analyzing/AST_to_IL.ml

```diff
@@ -685,6 +685,52 @@ and expr_aux env ?(void = false) e_gen =
         | _ -> exp
       in
       ident_function_call_hack exp
+  (* x = ClassName(args ...) in Python *)
+  (* ClassName has been resolved to __init__ by the pro engine. *)
+  (* Identified and treated as x = New ClassName(args ...) to support
```

Comment:
Well, also because the goal of the IL is also to make things "uniform" across languages. Add a `see HACK(new)` here so when reading this comment one can jump to the same code that we use for languages with explicit `new`.